router/customdestinationmanager/customdestinationmanager.go

```diff
@@ -283,18 +287,22 @@ func New(destType string, o Opts) DestinationManager {
 		}
```

Comment:
Forgive the silly question, but where are we writing into `customManagerMap`? :thinking: